test/test_helpers/pact_helper.rb

```diff
@@ -1,6 +1,7 @@
 PUBLISHING_API_PORT = 3001
 ORGANISATION_API_PORT = 3002
 BANK_HOLIDAYS_API_PORT = 3003
+ACCOUNT_API_PORT = 3004
```

Comment:
Interesting that they all need to be on different ports 🤔 Why is that?
Is this as a precaution in case we ever run more than two of these in the same test block in future?

Or does it complain about a port clash even if we're not pact testing ACCOUNT_API and BANK_HOLIDAYS_API in the same request?

Reply:
Pact throws an error if multiple services use the same port.  We might be able to fix that by moving the `Pact.service_consumer`s into the test files, but I think that's out of scope of the PR (and may be more complicated if we have to explicitly unregister the `service_consumer`s too).